vars/influxWriteData.groovy

```diff
@@ -84,7 +89,8 @@ void call(Map parameters = [:]) {
                 artifactVersion: script.commonPipelineEnvironment.getArtifactVersion(),
                 influxPrefix: script.commonPipelineEnvironment.getGithubOrg() && script.commonPipelineEnvironment.getGithubRepo()
                     ? "${script.commonPipelineEnvironment.getGithubOrg()}_${script.commonPipelineEnvironment.getGithubRepo()}"
-                    : null
+                    : null,
+                influxDbPluginClass: 'InfluxDbPublisher'
```

Comment:
shouldn't this rather be `addIfEmpty()`?
If I am not mistaken `InfluxDbPublisher` will now always overwrite the configuration.

@CCFenner what do you think?